{
  try {
    return invokeNextInterceptor(ctx,command);
  }
 catch (  Throwable throwable) {
    exceptionsCaught++;
    throw throwable;
  }
}
